dotCMS/src/main/java/com/dotmarketing/common/util/SQLUtil.java

```diff
@@ -237,6 +240,30 @@ public static String sanitizeSortBy(String parameter){
 		SecurityLogger.logDebug(SQLUtil.class, "Invalid or pernicious sql parameter passed in : " + parameter);
 		return StringPool.BLANK;
 	}
+
+	/**
+	 * Method to translate UI field into SQL columns naming
+	 * @param parameter
+	 * @return
+	 */
+	public  static String translateSortBy(String parameter){
+
+		String result = "";
+		result = parameter.replace("moddate","mod_date")

```

Comment:
I think it would be more efficient to use StringBuilder to avoid the generation of a new String any time

Reply:
but I still looking the String replacement
I would do something like that
```

public static void replace(final StringBuilder builder, final String from, final String to) {

    final int index = builder.indexOf(from);
    if (index != -1) {
        builder.replace(index, index + from.length(), to);
    }
}


final StringBuilder builder = new StringBuilder(parameter);

replace(builder, "moddate","mod_date");
```

You can add the replace on the MethodUtils and add an unit test for it